cmd/compute-domain-kubelet-plugin/nvlib.go

```diff
@@ -206,28 +206,41 @@ func (l deviceLib) getCliqueID() (string, error) {
 	uniqueCliqueIDs := make(map[string]struct{})
 
 	err := l.VisitDevices(func(i int, d nvdev.Device) error {
+		duid, ret := d.GetUUID()
+		if ret != nvml.SUCCESS {
+			return fmt.Errorf("failed to read device uuid (%d): %w", i, ret)
```

Comment:
This branch introduces one new error path in this function. For that reason, initially, I considered to only log this error and proceed (with a surrogate uuid). I after all decided not to do that, assuming that it's rather pathological and we in fact want to error out.

@elezar do we know of situations where `GetUUID()` failed and yet everything else seemed to be fine?